### ISSUE TITLE
LG-51: mpOpenTracing v1.3 - Implementation

### ISSUE THREAD

**Issue by fmhwong** [OpenLiberty/open-liberty#6338]
Aha: https://bigblue.aha.io/features/LG-51

ID: https://github.ibm.com/was-liberty/liberty-docs/issues/1057

Open Liberty Guide: https://github.com/OpenLiberty/guide-microprofile-opentracing/issues/63

Blog: https://github.com/openliberty/open-liberty/issues/6952

**Comment by marikaj123** [comment]
@fmhwong - Do you have English files for this feature that require translation?

**Comment by fmhwong** [comment]
@marikaj123 The message files that I have are directly copied from `mpOpenTracing-1.2`.  I copied the translated files too.  I don't have any new message.  In this case, do we still need to send the files to translation?

**Comment by marikaj123** [comment]
Please check the Master branch and let me know if it contains the expected translated files that you have copied.

**Comment by fmhwong** [comment]
Yes, it does. 
https://github.com/OpenLiberty/open-liberty/tree/master/dev/com.ibm.ws.opentracing.1.3/resources/com/ibm/ws/opentracing/resources

**Comment by fmhwong** [comment]
@skasund STE uploaded https://ibm.box.com/s/eq4edm76qiigzptq6n1osjevivdru6af

**Comment by fmhwong** [comment]
@gscottj This feature has no UI.

**Comment by fmhwong** [comment]
@shighbar This feature has no impact on CTS.

**Comment by skasund** [comment]
Thank you for the STE slides.  If L2 would like a live presentation, I'll have them reach out to you.

**Comment by gscottj** [comment]
Based on the description provided, it appears that this feature does not have any end-user user interfaces that require accessibility compliance testing. If this assumption is not correct, then let me know that required accessibility verification testing of the UI will be performed before it is shipped and what are the results from those tests.

**Comment by fmhwong** [comment]
@shighbar This feature has no impact on CTS.  May I get CTS approval?

**Comment by marikaj123** [comment]
@fmhwong - Features, in general, do not require CTS approval. Unless you know that your feature may impact the CTS results.

**Comment by fmhwong** [comment]
@jhanders Can you review the performance?  The tracing implementation of REST client is reusing the same filter from JAX-RS client.  The performance should be similar.

**Comment by fmhwong** [comment]
@bsbyrd1 ID issue has been created.  Can I have ID approval?  Thanks

**Comment by sabolo** [comment]
Please can you complete the feature test summary in order for the FAT review to proceed?  I have created the child issue and copied in the template for you.

**Comment by fmhwong** [comment]
@sabolo I have already created a feature test summary and tagged it.  It is #6795.

**Comment by fmhwong** [comment]
@chirp1 Karen, ID issue has been created. Can I have ID approval? Thanks

**Comment by marikaj123** [comment]
@fmhwong Please obtain WAD approval in AHA and then reflect it in this Epic. Thank you.

**Comment by fmhwong** [comment]
@marikaj123 I already created a WAD approval request in AHA and waiting for Alasdair to approve.

**Comment by fmhwong** [comment]
Adding [WAD Approved] as Alasdair approved it in AHA.

**Comment by fmhwong** [comment]
@dacleyra Thanks for completing SVT.  Did you discover any serviceability issues during your test?

**Comment by dacleyra** [comment]
No serviceability issues

**Comment by fmhwong** [comment]
@donbourne Can I get serviceability approval?

**Comment by fmhwong** [comment]
Feature is shipped in 19.0.0.3.
